taglib/ogg/flac/oggflacfile.cpp

```diff
@@ -233,7 +233,12 @@ void Ogg::FLAC::File::scan()
 
   }
 
-  header = metadataHeader.mid(0,4);
+  header = metadataHeader.mid(0, 4);
+  if(header.size() < 4) {
```

Comment:
Also, there is identical code just below. If we add the check here, it would be good to also add it there. But I wonder if it's possible to detect this kind of error earlier in Ogg packet parsing.